aicsimageio/tests/test_aics_image.py

```diff
@@ -1129,3 +1129,81 @@ def test_parallel_read(
     cluster.close()
     client.close()
     time.sleep(5)
+
+
+@pytest.mark.parametrize(
+    "filename, "
+    "host, "
+    "first_scene, "
+    "expected_first_chunk_shape, "
+    "second_scene, "
```

Comment:
FWIW I'm pretty sure the spaces at the end here are not needed (I usually only include them so separation on the same line is easier, but here that's redundant)

Reply:
I like it because it makes it very clear to me that this is a string continuation personally :shrug:

Reply:
Like I just do that for all strings throughtout like all of my repos. If the string is continuing I try to find spots to break the sentences by whitespace characters.